backend/organization/views/project_views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework.generics import ListAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.pagination import PageNumberPagination
+from rest_framework.filters import SearchFilter
+
+from organization.models import Project
+from organization.serializers.project import ProjectSerializer
+
+
+class ListProjectsView(ListAPIView):
+    permission_classes = [IsAuthenticated]
```

Comment:
We definitely need to show projects to users who aren't logged in. Here we could also do 2 different serializers for logged in and not logged in users.